test/common/grpc/BUILD

```diff
@@ -163,8 +163,6 @@ envoy_cc_test_library(
 envoy_cc_test(
     name = "grpc_client_integration_test",
     srcs = ["grpc_client_integration_test.cc"],
-    # Fails on windows; Connection is terminated early testing BasicStream/IPv4_EnvoyGrpc
-    tags = ["fails_on_windows"],
```

Comment:
Also remove the following test:

`//test/extensions/grpc_credentials/file_based_metadata:file_based_metadata_grpc_credentials_test`

Reply:
fixed in c17804639f521ca504056afb3506f8ef5aa85f80